components/compliance-service/api/tests/08_deep_search_profiles_spec.rb

```diff
@@ -124,7 +124,7 @@ def reporting;
                         },
                         "status" => "failed",
                         "controls" => {
-                            "total" => 1,
+                            "total" => 2,
```

Comment:
putting this 2 in here just to test that the pipeline will properly fail.  this should cause a test failure.  once this works, i'll change it back to 1